youtube_dl/extractor/dailymail.py

```diff
@@ -19,14 +20,17 @@ class DailyMailIE(InfoExtractor):
             'title': 'Turn any video into an impressionist masterpiece',
             'description': 'md5:88ddbcb504367987b2708bb38677c9d2',
         }
-    }
+    }, {
+        'url': 'www.dailymail.co.uk/video/tvshowbiz/video-1295863/The-Mountain-appears-sparkling-water-ad-Heavy-Bubbles.html',
+        'only_matching': True,
```

Comment:
It does not test the purpose of this change, i.e. title extraction improvement. It should replace the old test.